### ISSUE TITLE
Get prediction probabilities

### ISSUE THREAD

**Issue by DhruvAwasthi** [qubvel/segmentation_models.pytorch#389]
Is there a way to get prediction with probabilities?

**Comment by fabiofumarola** [comment]
net_output.softmax(1) ?
